js/apps/admin-ui/cypress/support/util/AdminClient.ts

```diff
@@ -352,6 +354,16 @@ class AdminClient {
       ),
     );
   }
+
+  async inRealm<T>(realm: string, fn: () => Promise<T>) {
+    const prevRealm = this.#client.realmName;
+    this.#client.realmName = realm;
+    try {
+      return await fn();
+    } finally {
```

Comment:
No need to `catch()` here?

Reply:
Should an exception be handled, it should be caught by the caller, since this code has no knowledge of the expectations of the exceptions.